### DISCUSSION TITLE
Automatically Downloading Website Images

### DISCUSSION THREAD

**Discussion by RABIDYOTE** [subhra74/xdm#1091] [Q&A]
I've noticed a weird issue I'm having with XDM. Sometimes, when I'm browsing certain sites (typically tech advice and help sites), XDM will automatically start downloading the page's images. Curiously, the page's images are also left blank, but this is fixed if I close XDM and delete the page's stored data. I don't have any image file formats typed and enabled in XDM's browser monitoring settings, so I'm not entirely sure what's causing this to happen. I am using the 8.0.26 beta version.

![Screenshot 2023-06-25 160139](https://github.com/subhra74/xdm/assets/78206431/55136e34-2e02-4da1-beb3-040369a9a06e)
![Screenshot 2023-06-25 155941](https://github.com/subhra74/xdm/assets/78206431/390aa24b-4a48-4df5-a8bb-111fe918f7be)

**Comment by ghost** [comment]
I'm also experiencing the same problem with XDM on Windows!

When browsing certain tech advice and help sites, XDM automatically downloads the page's images, even though they are left blank.

I also don't have any image file formats enabled in XDM's browser monitoring settings. This seems to be a bug with the 8.0.29 BETA version.

Hopefully, the developers can address this issue soon. If anyone finds a solution, please share it here!
